cmd/cerbosctl/audit/audit.go

```diff
@@ -226,6 +226,10 @@ func newRichAuditLogWriter(out io.Writer) *richAuditLogWriter {
 }
 
 func (r *richAuditLogWriter) write(entry proto.Message) error {
+	if entry == nil {
```

Comment:
I got a nil deref panic when running `cerbosctl audit` and I traced it here. It's a common Go pitfall: an interface holding a nil value is not nil. So, even if the underlying log entry is nil, this nil check will just let it through. 😄 

I think either you have to check for nil after the type switch below OR do the nil check in the `streamLogsToWriter` function above. My vote is for the latter because:
a) Not doing the nil check in `streamLogsToWriter` will result in two calls to `writer.write` per loop iteration with very little chance of it getting inlined or eliminated by the compiler -- therefore making it inefficient.
b) If the nil check is added after the type switch, you need to make the same change to `rawLogWriter.write` method as well. It's unnecessary duplication of code and still leaves the trap open for a future implementation of another `auditLogWriter` to accidentally omit the nil check and cause a panic.

Reply:
wow, this is.. 🤯